canvas_modules/common-canvas/__tests__/common-properties/properties-controller-test.js

```diff
@@ -29,6 +29,7 @@ import actionParamDef from "../test_resources/paramDefs/action_paramDef.json";
 import numberfieldParamDef from "../test_resources/paramDefs/numberfield_paramDef.json";
 import structuretablePropertyValues from "../test_resources/json/structuretable_propertyValues.json";
 import ExpressionInfo from "../test_resources/json/expression-function-list.json";
+import rejectLinkParamDe from "../test_resources/paramDefs/rejectLink_paramDef.json";
```

Comment:
This shouldn't be a new paramDef.  Can we add a new table or update a table in `readonlyTable_paramDef.json` for this?

Reply:
thanks.  I changed the name to hideEditButton and as not necessary to involve a new json, I use "readonlytable_paramDef.json" instead.